tests/integration/test_vulnerability_detector/test_feeds/alas/test_invalid_values_alas_feed.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
```

Comment:
Apply similar changes specified in previous tests

Reply:
Done in commit c37258d4f1e25c2c1bf06f8496f298d697ca6f0c